sdk/core/Azure.Core/src/Pipeline/Internal/RetryPolicy.cs

```diff
@@ -197,11 +260,39 @@ private void GetDelay(HttpMessage message, int attempted, out TimeSpan delay)
             {
                 delay = serverDelay;
             }
+
+            return delay;
         }
 
-        private void GetDelay(int attempted, out TimeSpan delay)
+        internal virtual TimeSpan GetServerDelay(HttpMessage message)
```

Comment:
This makes me think it's code running on the server - do we mean GetRetryAfterValue?

Reply:
Yeah that is what this is doing. This is an existing method - the diff is a little wonky here.